extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java

```diff
@@ -122,7 +125,12 @@ public void handleMessage(Object msg) {
                             responseBuilder.setBase64Encoded(true);
                             responseBuilder.setBody(Base64.getMimeEncoder().encodeToString(baos.toByteArray()));
                         } else {
-                            responseBuilder.setBody(new String(baos.toByteArray(), "UTF-8"));
+                            responseBuilder.setBody(new String(baos.toByteArray(), StandardCharsets.UTF_8));
+                        }
+                        try {
+                            baos.close();
```

Comment:
Again, this just makes the code less readable.  BAOS.close() doesn't do anything.

Reply:
Reverted